WordPress/Classes/Utility/WPUserAgent.h

```diff
@@ -6,6 +6,8 @@
  */
 @interface WPUserAgent : NSObject
 
++ (instancetype)sharedInstance;
```

Comment:
Since `WPUserAgent` doesn't hold any state, why do you need a shared instance? Can't you just instantiate a new one when needed?